packages/@ember/-internals/glimmer/lib/templates/empty.ts

```diff
@@ -0,0 +1,4 @@
+import { precompileTemplate } from '@ember/template-compilation';
+export default precompileTemplate('', {
+  moduleName: 'packages/@ember/-internals/glimmer/lib/templates/empty.hbs',
+});
```

Comment:
I was able to drop all special support for hbs files, and several hacks around their types, by making them just be the equivalent javascript.